drivers/hmis/app/models/hmis/hud/client.rb

```diff
@@ -95,6 +100,43 @@ def ssn_serial
     self.SSN&.[](-4..-1)
   end
 
+  def mci_external_id
+    external_ids_by_slug('clientview').first
```

Comment:
```suggestion
    external_ids_by_slug('mci').first&.value
```

Reply:
There are two remote credentials: One for storing the ClientView base url, which has slug "clientview." The second is for storing MCI auth credentials, which has slug "mci." The MCI External IDs themselves are linked to the "mci" remote credential. Sorry for the confusion!

My point with [this comment](https://github.com/greenriver/hmis-warehouse/pull/2981#discussion_r1172938222) is that you shouldn't actually use the `external_id.remote_credential` to find the link base. Since its a different remote credential, you can just look it up.

Reply:
Got it, thanks